server/src/prisma/statics/node.ts

```diff
@@ -0,0 +1,29 @@
+const findOrCreateMany = async (ids, nodes) => {
```

Comment:
to fix error : `this `might be undefined
```suggestion
async function findOrCreateMany(ids, nodes) {
```